environment/src/main/java/com/sequenceiq/environment/environment/validation/EnvironmentFlowValidatorService.java

```diff
@@ -35,7 +35,7 @@ public ValidationResult validateNetworkWithProvider(EnvironmentDto environmentDt
         return environmentNetworkProviderValidator.validate(environmentDto);
     }
 
-    public ValidationResult validateAndDetermineAwsParameters(EnvironmentDto environmentDto, AwsParametersDto awsParameters) {
-        return awsParameterValidator.validateAndDetermineAwsParameters(environmentDto, awsParameters);
+    public ValidationResult processAwsParameters(EnvironmentDto environmentDto, ParametersDto awsParameters) {
+        return awsParameterProcessor.processAwsParameters(environmentDto, awsParameters);
```

Comment:
awsParameters need to change to parametersDto

Reply:
alright